ISSUE_TEMPLATE.md

```diff
@@ -4,8 +4,10 @@
 - [ ] Issue Title is in form `Algo/DS Name [Language]`.
 - [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
 - [ ] This Algo/DS is worth adding to this repository.
+- [ ] I ensure that I'm not already working on 2 or more issues.
 - [ ] Assign this issue to me (I want to work on this).
 - [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
+- [ ] Unassign me if I fail to send Pull Request a week.
```

Comment:
Make it like `I understand that I will be unassigned from this issue if I fail...`